x-pack/plugins/integration_assistant/server/constants.ts

```diff
@@ -8,3 +8,4 @@
 export const ROUTE_HANDLER_TIMEOUT = 10 * 60 * 1000; // 10 * 60 seconds = 10 minutes
 export const LANG_CHAIN_TIMEOUT = ROUTE_HANDLER_TIMEOUT - 10_000; // 9 minutes 50 seconds
 export const CONNECTOR_TIMEOUT = LANG_CHAIN_TIMEOUT - 10_000; // 9 minutes 40 seconds
+export const MAX_IMPORT_PAYLOAD_BYTES = 9000000; // 9 MegaBytes
```

Comment:
This constant is never used, and this number is hardcoded in the UI. Can we move this constant to the UI and use it there? Or, if it also needs to be used by the server at some point, move it somewhere under `integration_assistant/common/*`, so it can be imported from both sides.

Reply:
Yep... Moved it to UI